client/state/stats/lists/selectors.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import get from 'lodash/get';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import i18n from 'lib/mixins/i18n';
```

Comment:
At one point in the distant past, we did do just that - but for some reason it was changed to utilize the mixin.  I believe it was just a standards thing, but there may have been another reason for it that I just don't remember :)